### PR TITLE
Add discovery support for zwave_js

### PR DESCRIPTION
**Pull request by pvizeli**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (which adds functionality to the supervisor)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code has been formatted using Black (`black --fast supervisor tests`)
- [x] Tests have been added to verify that the new code works.

If API endpoints of add-on configuration are added/changed:

- [ ] Documentation added/updated for [developers.home-assistant.io][docs-repository]

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[docs-repository]: https://github.com/home-assistant/developers.home-assistant

### CODE REVIEW

File: supervisor/discovery/services/ozw.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-"""Discovery service for Zwave MQTT."""
+"""Discovery service for OpenZwave MQTT."""
```
Comment (marcelveldt) [inline supervisor/discovery/services/ozw.py:1]: Discovery service for Z-Wave JS Server.

Comment (pvizeli) [reply supervisor/discovery/services/ozw.py:1]: Right, I updated also the files from the "old" integration for making the name clear

File: tests/discovery/test_zwave_js.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+"""Test Zwave MQTT discovery."""
```
Comment (marcelveldt) [inline tests/discovery/test_zwave_js.py:1]: """Test Z-Wave JS Server discovery."""

File: supervisor/discovery/services/ozw.py (hunk 1 above)
Comment (marcelveldt) [reply supervisor/discovery/services/ozw.py:1]: so there will be another service defined for zwave_js, right ?

Comment (marcelveldt) [reply supervisor/discovery/services/ozw.py:1]: All clear now. Changes to ozw are in the same commit. Ignore my ramblings then!

**Review by balloob — APPROVED** [review]

**Pull request merged**